### DISCUSSION TITLE
是否接受使用其他 EVM 相容鏈的技術文章

### DISCUSSION THREAD

**Discussion by gasolin** [lun-dao/LunDAO#47] [General]
他山之石可以攻錯，也許可以看到有些其他鏈的 EVM 實現也有啟發性？

**Comment by yurenju** [comment]
我覺得可以呀，應該是對整體有幫助的都沒問題，像是 Starknet 也不是 EVM 相容。其他鏈的知識如果有助於 Ethereum 社群發展我都覺得可以。

**Comment by hydai** [comment]
1. EVM 相容鏈的話，我會希望聚焦在他跟 EVM 之間的差異點、優劣處在哪，有沒有直接學習或參考的地方。
2. 甚至是 EVM 不相容的話，找到與 EVM 社群相關的連結，且能擴展對區塊鏈發展的廣度的話，我認為也可以納入考量。
